### DISCUSSION TITLE
PDF Portfolio

### DISCUSSION THREAD

**Discussion by joesalvati68** [QuestPDF/QuestPDF#503] [Q&A] [answered]
We need to create a pdf portfolio of multiple documents that are independent and hyperlinked by other processes that we are performing.
Can we use this library to create a PDF portfolio?
Thanks for the help.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out.

PDF Portfolio creation is not currently supported by QuestPDF. However, the `DocumentOperation` API does support merging multiple PDF documents and adding file attachments, which may partially cover your use case:
https://www.questpdf.com/concepts/document-operations.html

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
